tests/integration_tests/subtensor_client.py

```diff
@@ -0,0 +1,584 @@
+import asyncio
```

Comment:
Can we rename this to test_subtensor_client.py? makes it easier with other testing frameworks.